Sources/Core/Commons/Tools/Export+Write.swift

```diff
@@ -6,7 +6,7 @@
 //
 
 public extension Export {
-    func write() -> Result<Export, Xccov.Error> {
-        self.content.write(filename: self.filename).map { self }
+    func write(toDirectory directory: String) -> Result<Export, Xccov.Error> {
```

Comment:
in order to be consistent with the existing naming (and also the Apple naming), could you rename `toDirectory directory` in `atPath path` please ?

Reply:
done